pennylane_qiskit/qiskit_device2.py

```diff
@@ -288,8 +288,7 @@ def __init__(
         self._backend = backend
         self._compile_backend = compile_backend if compile_backend else backend
 
-        # ToDo: possibly things fail if this is not a QiskitRuntimeService - confirm and decide how to handle (SC 55725)
-        self._service = backend._service
+        self._service = getattr(backend, "_service", None)
```

Comment:
What happens with the fallback `None` value? Do we use it to raise some intuitive error?

Reply:
If service is `None` it means that the backend being used is a local simulator. No need for an error.